network/p2p/p2pnode/libp2pNode.go

```diff
@@ -90,8 +90,6 @@ func NewNode(
 	}
 }
 
-var _ p2p.LibP2PNode = (*Node)(nil)
```

Comment:
Why was this removed?

Reply:
its already declared here https://github.com/onflow/flow-go/blob/9c5b9f154064b543b01031a82afbd1c7846ead5b/network/p2p/p2pnode/libp2pNode.go#L51

Reply:
@nozim, kindly allow the reviewer, in this case @kc1116, to resolve the conversation. When the PR author unilaterally resolves a conversation, it complicates the tracking of the PR's progress.

Reply:
my apologies @yhassanzadeh13 and @kc1116. Will let the reviewer resolve future feedback.